.github/workflows/CrossRefCheckWithAnchors.yml

```diff
@@ -0,0 +1,12 @@
+name: CrossRefCheck with Anchors
+
+on: 
+  push:
+    branches:
+      - review
+
+jobs:
+  crossref-check-with-anchors:
+    uses: Open-Systems-Pharmacology/Workflows/.github/workflows/MarkdownCrossrefCheck_withAnchors.yml@main
+    with:
+      folder-path: 'Evaluation/report'
```

Comment:
Check of internal references incl. anchors. 
Will be done only in the **review** branch (because the report should not be uploaded in the XXX-Model repo).

The check is done again for all Evaluation Reports in the [OSP-PBPK-Model-Library Repo](https://github.com/Open-Systems-Pharmacology/OSP-PBPK-Model-Library/tree/master/.github/workflows) - but it's better to check early before you create a release :)